static/css/style.scss

```diff
@@ -199,107 +201,128 @@ footer {
 	.faq-section {
 		margin-bottom: 45px;
 	}
+
 	h2 {
 		margin-top: 0;
 	}
+
 	h3 {
-		font-size:1em;
+		font-size: 1em;
+
 		//color: Silver  ;
-		margin-top:1.5em;
+		margin-top: 1.5em;
 	}
 }
+
 // table of contents for faq page
 
 #toc {
-	display:block;
+	display: block;
 	position: relative;
 	margin-bottom: 3em;
+
 	nav {
-		top:15px;
+		top: 15px;
 	}
-	.nav>li>a:hover, .nav>li>a:focus {
-		background:none;
+
+	.nav > li > a:hover, .nav > li > a:focus {
+		background: none;
 	}
+
 	li.active a {
-		font-weight:bold;
+		font-weight: bold;
+
 		//color:#000;
+
 	}
+
 	li a:hover {
-		background:none;
-		text-decoration:underline;
+		background: none;
+		text-decoration: underline;
 	}
+
 	.toc-h2 a {
 		//color: black;
-		font-weight:bold;
-		padding:0;
+		font-weight: bold;
+		padding: 0;
 	}
-	.toc-h3 + .toc-h2  {
-		margin-top:1em;
+
+	.toc-h3 + .toc-h2 {
+		margin-top: 1em;
 	}
+
 	.toc-h3 a {
-		color: Silver  ;
+		color: Silver;
+
 		//padding-left:1em;
-		padding:0;
+		padding: 0;
 	}
+
 	a {
-		padding:0;
+		padding: 0;
 	}
 }
 
 // filter buttons for people page
 
 #filters {
-	text-align:center;
+	text-align: center;
 	margin-bottom: 15px;
+
 	.btn-group {
 		margin-bottom: 15px;
 	}
+
 	button {
 		outline: 0;
 	}
-
 }
 
 // people page grid styles
 
 ul#people-grid {
-	list-style-type:none;
-	margin:0;
-	padding:0;
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+
 	li {
-		margin-bottom:15px;
+		margin-bottom: 15px;
 	}
+
 	.info {
-		position:relative;
-		display:block;
-		width:100%;
+		position: relative;
+		display: block;
+		width: 100%;
 	}
+
 	.links {
-		opacity:0;
+		opacity: 0;
 		-webkit-transition: opacity .15s;
 		transition: opacity .5s;
-		text-align:right;
-		width:50%;
-		position:absolute;
-			top:1px;
-			right:0;
-		background: -moz-linear-gradient(left, rgba(255,255,255,0) 0%, rgba(255,255,255,1) 15%, rgba(255,255,255,1) 100%); /* FF3.6+ */
-		background: -webkit-gradient(linear, left top, right top, color-stop(0%,rgba(255,255,255,0)), color-stop(15%,rgba(255,255,255,1)), color-stop(100%,rgba(255,255,255,1))); /* Chrome,Safari4+ */
-		background: -webkit-linear-gradient(left, rgba(255,255,255,0) 0%,rgba(255,255,255,1) 15%,rgba(255,255,255,1) 100%); /* Chrome10+,Safari5.1+ */
-		background: -o-linear-gradient(left, rgba(255,255,255,0) 0%,rgba(255,255,255,1) 15%,rgba(255,255,255,1) 100%); /* Opera 11.10+ */
-		background: -ms-linear-gradient(left, rgba(255,255,255,0) 0%,rgba(255,255,255,1) 15%,rgba(255,255,255,1) 100%); /* IE10+ */
-		background: linear-gradient(to right, rgba(255,255,255,0) 0%,rgba(255,255,255,1) 15%,rgba(255,255,255,1) 100%); /* W3C */
+		text-align: right;
+		width: 50%;
+		position: absolute;
+		top: 1px;
+		right: 0;
+		background: -moz-linear-gradient(left, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 1) 15%, rgba(255, 255, 255, 1) 100%); /* FF3.6+ */
```

Comment:
Not a blocker, but some of these prefixed properties aren't needed anymore. It would be nice to get [autoprefixer](http://georgeowen.co.uk/blog/autoprefixer-jekyll/) set up to avoid having to manually write / check this.